toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -644,6 +669,7 @@ func calculateTotalPackages(packages []string, installRoot string) (totalPackage
 	}
 
 	// For every package calculate what dependencies would also be installed from it.
+	foundPackages := make(map[string]bool)
```

Comment:
nit: a comment on what does foundPackages mean would be helpful

Reply:
I've been trying to name the `map[<TYPE>]bool` variables we use everywhere `...Set` to help explain their expected use. Maybe `checkedPackageSet`?

Reply:
sure thing, updated